tests/unit/utils/txn_parser/transaction_jsons/payment_token.json

```diff
@@ -0,0 +1,123 @@
+{
```

Comment:
nit: I'd call this `payment_iou.json`, since we mainly call tokens IOUs internally to the repo